File: Models/DB/Recipe.cs
using System.Collections.Generic;

namespace mijn_recepten.Models.DB {
    public class Recipe {
        public int Id {get; set;}
        public string Name {get; set;}
        public string Requester {get; set;}
        public string VideoId {get; set;}
        public string Persons {get; set;}
        public List<Link> Links {get; set;}
        public List<Ingredient> Ingredients {get; set;}
    }
}

File: ClientApp/src/app/_models/common/recipe.ts
import { Ingredients } from './ingredients';
import { Links } from './links';

export class Recipe {
    name: string;
    requester: string;
    persons: string;
    id: number;
    ingredients: Ingredients[];
    links: Links[];
    favorite: boolean;
    videoId: string;

    constructor(name: string, requester: string, persons: string, ingredients: Ingredients[], links: Links[], videoId: string) {
        this.name = name;
        this.requester = requester;
        this.persons = persons;
        this.ingredients = ingredients;
        this.links = links;
        this.videoId = videoId;
        this.id = 0;
    }
}
